llvm/lib/Transforms/Utils/LoopIterCountAssumptions.cpp

```diff
@@ -205,7 +205,7 @@ std::optional<int64_t> getMinIterCount(Loop &CurrentLoop) {
   if (!CurrentLoop.isRotatedForm())
     return RawMinIterationCount;
 
-  const int BackedgeCount = MinIterCount - 1;
```

Comment:
Did that actually cause an issue?

Reply:
issue in the sense of performance regression or casting issue?